<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>889</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="plot_groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>841</width>
     <height>431</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>4777215</width>
     <height>4777215</height>
    </size>
   </property>
   <property name="title">
    <string>Calibration</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QGroupBox" name="Live_groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>851</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string>Settings</string>
   </property>
   <widget class="QLabel" name="label_min">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>23</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>white level</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_max">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>black level</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="min_doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>53</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="autoLevels_checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>75</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>autoLevels</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="autoRange_checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>109</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>autoRange</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="max_doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>53</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="pattern_groupBox_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>331</width>
      <height>401</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>4777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string>Camera</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="16" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>frame_rate</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>sensor_mode</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>exposure_time</string>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QDoubleSpinBox" name="internal_frame_interval_doubleSpinBox"/>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>effective_frame_exposure</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QComboBox" name="sensor_mode_comboBox">
       <item>
        <property name="text">
         <string>area</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>progressive</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>readout_direction</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QDoubleSpinBox" name="internal_line_interval_doubleSpinBox"/>
     </item>
     <item row="17" column="0">
      <widget class="QDoubleSpinBox" name="internal_frame_rate_doubleSpinBox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="0">
      <widget class="QDoubleSpinBox" name="exposure_time_doubleSpinBox"/>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="connected_checkBox">
       <property name="text">
        <string>Connected</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QComboBox" name="readout_direction_comboBox">
       <item>
        <property name="text">
         <string>backward</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>forward</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>readout_time</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progress_progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="PI_groupBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>341</width>
      <height>401</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>4777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string>PI parameters</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>pp_amplitude</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="connected_pi_checkBox">
       <property name="text">
        <string>Connected</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>starting_position</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="0">
      <widget class="QTextBrowser" name="textBrowser_2">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Positive amplitude &amp;lt;---&amp;gt; Forward &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Negative amplitude &amp;lt;---&amp;gt; Backward &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="set_home_pushButton">
       <property name="text">
        <string>set_home</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QDoubleSpinBox" name="starting_position_doubleSpinBox"/>
     </item>
     <item row="7" column="0">
      <widget class="QDoubleSpinBox" name="pp_amplitude_doubleSpinBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="rotate_image_comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>90</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>180</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>270</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>rotate_image</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>171</width>
      <height>181</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="start_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Calibration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="interrupt_pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Interrupt</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
